### DISCUSSION TITLE
What files can be deleted when publishing an unpackaged app?

### DISCUSSION THREAD

**Discussion by AzAgarampur** [microsoft/WindowsAppSDK#2001] [Q&A] [answered]
After building my unpackaged app in Release, there are some files in the release directory that seem to be used only when creating MSIX packages, not unpackaged apps. I already deleted the .pdb and linker files because they are unnecessary, but what about the .winmd files and .pri files?

For a test, create an unpackaged app using the directions on microsoft docs. Build in release, then remove the pdb and extra linker files. Then, delete everything except the .XBF files. Run the program, and notice how it runs the same compared to if the files were present.

Is this safe to do, and in what situations will we need those other files?

On the other hand, is there a way for .XBFs to be merged into our app the same way they are merged if you are publishing a C++/WinRT or C# UWP app?

**Comment by BenJKuhn** [comment, marked as answer]
winmd files are used at runtime sometimes, and should be kept as well as a general rule. We should provide clear documentation on this topic though. I've opened this as a documentation issue.

https://github.com/microsoft/WindowsAppSDK/issues/2006

Ben
